web/src/main/resources/org/akaza/openclinica/i18n/page_messages.properties

```diff
@@ -1077,15 +1060,11 @@ right_item_length_error=Right item text cannot be more than 4000 characters
 item_header_length_error=Item header cannot be more than 2000 characters
 item_subheader_length_error=Item subheader cannot be more than 240 characters
 regexp_errror_length_error=Regexp error cannot be more than 255 characters
-group_label_length_error=Group Label cannot be more than 255 characters
```

Comment:
This string is still referenced in SpreadSheetTableRepeating.java.  Should not be deleted.